tests/generic_config_updater/test_eth_interface.py

```diff
@@ -56,18 +58,39 @@ def is_valid_fec_state_db(duthost, value):
     return True
 
 
-def is_valid_speed_state_db(duthost, value):
-    read_supported_speeds_cli = 'sonic-db-cli STATE_DB hget "PORT_TABLE|{}" supported_speeds'.format("Ethernet0")
```

Comment:
>Ethernet0 [](http://example.com/codeflow?start=102&length=9)

This seems a valid bugfix, we cannot assume Ethernet0 always exists in every hwsku. Does it make sense to submit a separate PR for this bugfix?